composer.lock

```diff
@@ -151,24 +151,24 @@
         },
         {
             "name": "composer/semver",
-            "version": "3.4.2",
```

Comment:
should there be accompanying composer.json changes here? e.g. how does composer.lock get updated without composer.json changes?

Reply:
I did a composer update but there were no composer.json changes only changes in the lock file

Reply:
I see - that probably makes sense!